parquet/src/test/java/org/apache/iceberg/parquet/TestParquetMetrics.java

```diff
@@ -292,14 +290,6 @@ private void assertCounts(int fieldId, long valueCount, long nullValueCount, Met
   }
 
   private File writeRecords(Schema schema, Record... records) throws IOException {
-    File tmpFolder = temp.newFolder("parquet");
-    String filename = UUID.randomUUID().toString();
-    File file = new File(tmpFolder, FileFormat.PARQUET.addExtension(filename));
-    try (FileAppender<Record> writer = Parquet.write(localOutput(file))
-        .schema(schema)
-        .build()) {
-      writer.addAll(Lists.newArrayList(records));
-    }
-    return file;
+    return TestParquet.writeRecords(temp, schema, Collections.emptyMap(), null, records);
```

Comment:
Tests shouldn't use methods in other tests because they are hard to keep track of. If you want to use the same method, then let's move it to a test utility class, or a base class.

Reply:
Did a base class here just for convenience so the `@Rule` for the temp folder doesn't need to be passed around.